pkg/scheduler/schedule_one_test.go

```diff
@@ -1746,7 +1746,13 @@ func TestFindNodesThatPassExtenders(t *testing.T) {
 			}
 
 			pod := st.MakePod().Name("1").UID("1").Obj()
-			got, err := findNodesThatPassExtenders(ctx, extenders, pod, tt.nodes, tt.filteredNodesStatuses)
+			got, err := findNodesThatPassExtenders(ctx, extenders, pod, tf.BuildNodeInfos(tt.nodes), tt.filteredNodesStatuses)
+
+			nodes := make([]*v1.Node, len(got))
```

Comment:
Is this the only case where we have this loop?

Reply:
Yes, `findNodesThatPassExtenders`'s return values got changed. So we change it's return value back to node in test for convenient.

Although there are other functions which have the same change in return value. We don't seem to have tests for these functions(like `findNodesThatPassFilters`).

So, currently, this is the only case.